homeassistant/components/media_player/plex.py

```diff
@@ -192,6 +200,18 @@ def update_devices():
             client.set_availability(client.machine_identifier
                                     in available_client_ids)
 
+            if not config.get(CONF_REMOVE_UNAVAILABLE_CLIENTS) \
+                    or client.available:
+                continue
+
+            if (dt.now() - client.marked_unavailable) >= \
```

Comment:
Don't use `datetime.now()`, instead use our dateutil package and the `dt_util.utcnow()` method:

```python
from homeassistant.util import dt as dt_util
```

Reply:
What is the difference?

Reply:
UTC time doesn't do daylight savings